features/03_testing_frameworks/cucumber/steps/filesystem/check_file_content.feature

```diff
@@ -3,6 +3,63 @@ Feature: Check file content
   Background:
     Given I use a fixture named "cli-app"
 
+  Scenario: Check file contents with plain text
+    Given a file named "features/non-existence.feature" with:
+    """
+    Feature: Check
+      Scenario: Check
+        Given a file named "foo" with:
+        \"\"\"
```

Comment:
Are these escaped because were inside a `"""` already? I don't use `aruba` really.

Reply:
Yes, that is the reason. It's a little ugly :disappointed:.